docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -26,9 +34,12 @@ const PropTypeDescription = React.createClass({
   render() {
     const {
       code,
+      component,
+      description,
     } = this.props;
 
-    let text = `### Properties
```

Comment:
Following what we were saying. I'm wondering if we shouldn't do the opposite:
Optionally rendering the `### Properties`